react-accordion-component/my-app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import Accordion from './Accordion';
+import './App.css';
+import { useState } from 'react';
+
+function App() {
+  const [view, setView] = useState('none');
```

Comment:
I think you pulled this state up too high. The App doesn't need to know which view is visible. The Accordion can keep track of that easily enough.

Think of an Accordion as a component that displays a list of items and lets the user open and close them. It manages everything necessary to do that. All it needs is the list of items to display.

As with your AppDrawer, by putting the open state here, you put too big of a burden on the Accordion's parent.